### PR TITLE
[#133] Regression tests are executed two times faster

### PR DESCRIPTION
**Pull request by ablatov**
- pabot library integration for parallel execution https://github.com/mkorpela/pabot.
- all tests are separated to 4 suites (4 processes).
- the third test-bare-model added.
- jenkins_dex_client updated for the cookies catching before all processes start.
- tests execution time changed from 40-50 minutes to 15-20 minutes.

### CODE REVIEW

**Comment by ablatov** [comment]
ok to test

**Comment by ablatov** [comment]
ok to test

**Comment by kirillmakhonin** [comment]
@ablatov please update this branch due to merged #438

**Comment by ablatov** [comment]
ok to test

**Comment by dsuslov** [comment]
test this please

**Comment by dsuslov** [comment]
ok to test

**Comment by dsuslov** [comment]
ok to test

**Comment by kirillmakhonin** [comment]
ok to test

**Comment by dsuslov** [comment]
test this please

**Comment by ablatov** [comment]
ok to test

File: k8s/test-bare-model-api/model-3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.6-alpine3.7
```
[the remark this answers is not in the document]
Comment (ablatov) [reply k8s/test-bare-model-api/model-3/Dockerfile]: @aliaksandr-d done.

File: k8s/test-bare-model-api/model-3/app.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from flask import Flask, jsonify
```
[the remark this answers is not in the document]
Comment (ablatov) [reply k8s/test-bare-model-api/model-3/app.py]: @aliaksandr-d done.

**Comment by ablatov** [comment]
ok to test

**Comment by ablatov** [comment]
bot, test all

**Comment by ablatov** [comment]
bot, test all

**Comment by ablatov** [comment]
bot, test all

**Comment by ablatov** [comment]
bot, test all

**Comment by ablatov** [comment]
bot, test all

**Pull request merged**